src/client/fuse_ll.cc

```diff
@@ -389,13 +389,14 @@ static void fuse_ll_getattr(fuse_req_t req, fuse_ino_t ino,
 
   (void) fi; // XXX
 
-  if (cfuse->client->ll_getattr(in, &stbuf, perms)
-      == 0) {
+  int r = cfuse->client->ll_getattr(in, &stbuf, perms);
+  if (r == 0) {
     stbuf.st_ino = cfuse->make_fake_ino(stbuf.st_ino, stbuf.st_dev);
     stbuf.st_rdev = new_encode_dev(stbuf.st_rdev);
     fuse_reply_attr(req, &stbuf, 0);
   } else
-    fuse_reply_err(req, ENOENT);
+    fuse_reply_err(req, get_sys_errno(-r));
```

Comment:
Fair enough. Though I think why it was kept this way especially since get_sys_errno() is used everywhere else in fuse_ll.cc